### PR TITLE
SOS v1.3.4 release updates

### PR DESCRIPTION
**Pull request by jdinan**
* Update NEWS
* Bump version number in configure.ac

### CODE REVIEW

**Comment by jdinan** [comment]
Portals 4 build cleared testing on the local cluster.

**Comment by jdinan** [comment]
@matcabral Want to kick the tires on this before we tag it?

**Comment by matcabral** [comment]
sure,

**Comment by matcabral** [comment]
I see there has not been any commits after FI_MR_RMA_EVENT was implemented. So, this means this is passing all my tests. Good to go! :checkered_flag:

File: NEWS [hunk 1]
```diff
@@ -2,6 +2,21 @@
...
+  and wait polling limits, respectively.
+- Rewrite of environment variable management code and improved info output.
+- Added support for processor atomics in thread-safe build.
+- Added support for local communication optimization (--enable-memcpy).
+- Added support for libfabric FI_MR_RMA_EVENT memory registration mode.
+- Fixed completion race in multithreaded mode.
+- Fixed bugs in support for zero-length point-to-point and collective
+  communication.
+- Multiple bugfixes to test suite, including GUPS benchmark
+
```
Comment (davidozog) [inline NEWS]: Looks great - a few extras that may or may not warrant inclusion:
PMIx/OpenMPI launcher? (but, it's experimental)
Mellanox tests? (but, not user-facing...)

**Pull request merged**